lib/src/sv_sign.c

```diff
@@ -860,6 +862,16 @@ SignedVideoReturnCode
 signed_video_set_private_key(signed_video_t *self, const char *private_key, size_t private_key_size)
 {
   if (!self || !private_key || private_key_size == 0) return SV_INVALID_PARAMETER;
+#if defined(SIGNED_VIDEO_DEBUG)
+#endif
+  // If ONVIF is available, call its function and map the return code
+  if (self->onvif) {
+    const char *certificate_chain = get_certificate_chain(self);
```

Comment:
Since this is inside the if, `certificate_chain` cannot be NULL, but it is probably a good idea to check that with an assert.